hw/hw_04_task_02.py

```diff
@@ -0,0 +1,11 @@
+import urllib.request
+
+
+# flake8: noqa: S310
+def count_dots_on_i(url: str) -> int:
+    try:
+        page = urllib.request.urlopen(url)
+        data = page.read().decode()
+        return sum(1 for symb in data if symb == "i")
+    except ValueError:
```

Comment:
Why value error?  https://docs.python.org/3/library/exceptions.html?highlight=valueerror#ValueError

I assume that you haven't run this code with different URLs.  I'd recommend reproducing 404.